src/DonationForms/resources/components/Pagination.js

```diff
@@ -19,86 +20,89 @@ const Pagination = ({currentPage, totalPages, disabled, setPage}) => {
     const previousPage = parseInt(currentPage) - 1;
 
     return (
-        <div className="tablenav bottom">
-            <div className="tablenav-pages">
-                <div className="pagination-links">
-                    {previousPage > 0 ? (
-                        <>
-                            <a
-                                href="#"
-                                className="tablenav-pages-navspan button"
-                                onClick={(e) => {
-                                    e.preventDefault();
-                                    if (!disabled) {
-                                        setPage(1);
-                                    }
-                                }}
-                            >
-                                «
-                            </a>{' '}
-                            <a
-                                href="#"
-                                className="tablenav-pages-navspan button"
-                                onClick={(e) => {
-                                    e.preventDefault();
-                                    if (!disabled) {
-                                        setPage(parseInt(currentPage) - 1);
-                                    }
-                                }}
-                            >
-                                ‹
-                            </a>
-                        </>
-                    ) : (
-                        <span className="tablenav-pages-navspan button disabled">‹</span>
-                    )}
+        <nav className={styles.container}>
+            {previousPage > 0 ? (
+                <>
+                    <a
+                        href="#"
+                        className={cx(styles.navDirection, styles.navElement)}
+                        onClick={(e) => {
+                            e.preventDefault();
+                            if (!disabled) {
+                                setPage(1);
+                            }
+                        }}
+                    >
+                        «
+                    </a>{' '}
+                    <a
+                        href="#"
+                        className={cx(styles.navDirection, styles.navElement)}
+                        onClick={(e) => {
+                            e.preventDefault();
+                            if (!disabled) {
+                                setPage(parseInt(currentPage) - 1);
+                            }
+                        }}
+                    >
+                        ‹
+                    </a>
```

Comment:
Let’s make these `<button type="button">` elements since we are preventing the browser default and we aren’t doing anything meaningful to the URL anyway. If we were setting a URL search param, I would keep these as `<a>` elements, but we can cross that bridge when/if we get to it.